src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/Modals/CreatePasswordModal.js

```diff
@@ -16,7 +17,7 @@ const CreatePasswordModalBody = ({ email }) => {
       <Row>
         <Col
           xs={12}
-          style={{ fontWeight: 'bold', padding: '10px' }}
```

Comment:
Could we just simplify this whole thing to the following?
```
<Layout class="padding-all-gutter"><strong>{email}</strong></Layout>
```

We don't even need the Row and Col since it's always 12 columns wide.